An unhandled exception is being displayed to the end user
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "app/sensors/views.py", line 58, in csv_route
    
  File "app/sensors/views.py", line 58, in csv_route
    
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/bdb.py", line 49, in trace_dispatch
    return self.dispatch_line(frame)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/bdb.py", line 68, in dispatch_line
    if self.quitting: raise BdbQuit
BdbQuit
An unhandled exception is being displayed to the end user
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "app/sensors/views.py", line 85, in csv_route
    
  File "app/sensors/views.py", line 85, in csv_route
    
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/bdb.py", line 49, in trace_dispatch
    return self.dispatch_line(frame)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/bdb.py", line 68, in dispatch_line
    if self.quitting: raise BdbQuit
BdbQuit
An unhandled exception is being displayed to the end user
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "app/sensors/views.py", line 87, in csv_route
    print e
  File "app/sensors/views.py", line 87, in csv_route
    print e
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/bdb.py", line 49, in trace_dispatch
    return self.dispatch_line(frame)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/bdb.py", line 68, in dispatch_line
    if self.quitting: raise BdbQuit
BdbQuit
An unhandled exception is being displayed to the end user
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "app/sensors/views.py", line 87, in csv_route
    print e
  File "app/sensors/views.py", line 87, in csv_route
    print e
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/bdb.py", line 49, in trace_dispatch
    return self.dispatch_line(frame)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/bdb.py", line 68, in dispatch_line
    if self.quitting: raise BdbQuit
BdbQuit
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Changing label in on a db object...
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Changing label in on a db object...
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Committing 253 records to the database
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Error changing timestamp time data 'fjdj' does not match format '%Y-%m-%d %H:%M:%S:%f'
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Changing label in on a db object...
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Error changing timestamp time data 'wtfwtf' does not match format '%Y-%m-%d %H:%M:%S:%f'
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Error changing timestamp time data 'wtfwtf' does not match format '%Y-%m-%d %H:%M:%S:%f'
Error changing timestamp time data 'fjdj' does not match format '%Y-%m-%d %H:%M:%S:%f'
Committing 253 records to the database
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Changing label in on a db object...
Error changing timestamp time data 'testing2' does not match format '%Y-%m-%d %H:%M:%S:%f'
Error changing timestamp time data 'testing3' does not match format '%Y-%m-%d %H:%M:%S:%f'
Error changing timestamp must be string, not float
Error changing timestamp must be string, not float
Error changing timestamp must be string, not float
Error changing timestamp must be string, not float
Error changing timestamp must be string, not float
Error changing timestamp must be string, not float
Error changing timestamp must be string, not float
Committing 9 records to the database
Error changing timestamp time data 'testing2' does not match format '%Y-%m-%d %H:%M:%S:%f'
Error changing timestamp time data 'testing3' does not match format '%Y-%m-%d %H:%M:%S:%f'
Committing 2 records to the database
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Request resulted in 404: Not Found
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
NotFound: 404: Not Found
Changing label in on a db object...
Request resulted in 405: Method Not Allowed
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
MethodNotAllowed: 405: Method Not Allowed
Request resulted in 405: Method Not Allowed
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
MethodNotAllowed: 405: Method Not Allowed
Request resulted in 405: Method Not Allowed
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
MethodNotAllowed: 405: Method Not Allowed
Request resulted in 405: Method Not Allowed
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
MethodNotAllowed: 405: Method Not Allowed
Request resulted in 405: Method Not Allowed
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
MethodNotAllowed: 405: Method Not Allowed
Request resulted in 405: Method Not Allowed
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
MethodNotAllowed: 405: Method Not Allowed
Changing label in on a db object...
Request resulted in 405: Method Not Allowed
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
MethodNotAllowed: 405: Method Not Allowed
Request resulted in 405: Method Not Allowed
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
MethodNotAllowed: 405: Method Not Allowed
Error changing timestamp time data 'wtfwtf' does not match format '%Y-%m-%d %H:%M:%S:%f'
Error changing timestamp time data 'fjdj' does not match format '%Y-%m-%d %H:%M:%S:%f'
Committing 253 records to the database
Committing 16 records to the database
Changing label in on a db object...
Changing label in on a db object...
Error changing timestamp time data 'testing2' does not match format '%Y-%m-%d %H:%M:%S:%f'
Error changing timestamp time data 'testing3' does not match format '%Y-%m-%d %H:%M:%S:%f'
Committing 2 records to the database
Changing label in on a db object...
Request resulted in 405: Method Not Allowed
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
MethodNotAllowed: 405: Method Not Allowed
Request resulted in 405: Method Not Allowed
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
MethodNotAllowed: 405: Method Not Allowed
Request resulted in 405: Method Not Allowed
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
MethodNotAllowed: 405: Method Not Allowed
Request resulted in 405: Method Not Allowed
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
MethodNotAllowed: 405: Method Not Allowed
Request resulted in 405: Method Not Allowed
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
MethodNotAllowed: 405: Method Not Allowed
Request resulted in 405: Method Not Allowed
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
MethodNotAllowed: 405: Method Not Allowed
Request resulted in 405: Method Not Allowed
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
MethodNotAllowed: 405: Method Not Allowed
Request resulted in 405: Method Not Allowed
Traceback (most recent call last):
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1453, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/christophersamiullah/repos/sensor_readings/venv/lib/python2.7/site-packages/flask/app.py", line 1436, in raise_routing_exception
    raise request.routing_exception
MethodNotAllowed: 405: Method Not Allowed
____api received_____ ------WebKitFormBoundaryu9S9G1KXB3HDtkBs
Content-Disposition: form-data; name="file"; filename="mbient7.csv"
Content-Type: text/csv

accelerometer_x,accelerometer_y,accelerometer_z,timestamp,
0.651,0.144,0.715,wtfwtf,
0.661,0.143,0.723,fjdj,
0.664,0.146,0.716,1899-12-30 00:00:00:120,
0.67,0.149,0.71,1899-12-30 00:00:00:140,
0.675,0.148,0.703,1899-12-30 00:00:00:160,
0.675,0.152,0.699,1899-12-30 00:00:00:180,
0.669,0.153,0.698,1899-12-30 00:00:00:200,
0.67,0.149,0.707,1899-12-30 00:00:00:220,
0.668,0.147,0.712,1899-12-30 00:00:00:240,
0.663,0.151,0.708,1899-12-30 00:00:00:260,
0.661,0.153,0.706,1899-12-30 00:00:00:280,
0.657,0.154,0.708,1899-12-30 00:00:00:300,
0.66,0.156,0.705,1899-12-30 00:00:00:320,
0.666,0.156,0.703,1899-12-30 00:00:00:340,
0.669,0.155,0.702,1899-12-30 00:00:00:360,
0.666,0.152,0.703,1899-12-30 00:00:00:380,
0.667,0.142,0.712,1899-12-30 00:00:00:400,
0.661,0.154,0.712,1899-12-30 00:00:00:420,
0.655,0.16,0.705,1899-12-30 00:00:00:440,
0.658,0.149,0.71,1899-12-30 00:00:00:460,
0.662,0.146,0.718,1899-12-30 00:00:00:480,
0.658,0.151,0.715,1899-12-30 00:00:00:500,
0.658,0.165,0.708,1899-12-30 00:00:00:520,
0.672,0.159,0.71,1899-12-30 00:00:00:540,
0.678,0.161,0.71,1899-12-30 00:00:00:560,
0.681,0.167,0.714,1899-12-30 00:00:00:580,
0.681,0.166,0.719,1899-12-30 00:00:00:600,
0.68,0.165,0.718,1899-12-30 00:00:00:620,
0.668,0.17,0.72,1899-12-30 00:00:00:640,
0.658,0.181,0.719,1899-12-30 00:00:00:660,
0.673,0.181,0.718,1899-12-30 00:00:00:680,
0.688,0.181,0.723,1899-12-30 00:00:00:700,
0.681,0.189,0.726,1899-12-30 00:00:00:720,
0.674,0.197,0.724,1899-12-30 00:00:00:740,
0.67,0.207,0.721,1899-12-30 00:00:00:760,
0.662,0.205,0.723,1899-12-30 00:00:00:780,
0.659,0.208,0.719,1899-12-30 00:00:00:800,
0.654,0.209,0.719,1899-12-30 00:00:00:820,
0.644,0.213,0.715,1899-12-30 00:00:00:840,
0.635,0.209,0.711,1899-12-30 00:00:00:860,
0.62,0.205,0.714,1899-12-30 00:00:00:880,
0.629,0.196,0.714,1899-12-30 00:00:00:900,
0.649,0.193,0.703,1899-12-30 00:00:00:920,
0.659,0.198,0.69,1899-12-30 00:00:00:940,
0.659,0.201,0.682,1899-12-30 00:00:00:960,
0.662,0.207,0.674,1899-12-30 00:00:00:980,
0.675,0.218,0.66,1899-12-30 00:00:01:000,
0.674,0.226,0.655,1899-12-30 00:00:01:020,
0.669,0.223,0.668,1899-12-30 00:00:01:040,
0.674,0.231,0.667,1899-12-30 00:00:01:060,
0.685,0.235,0.662,1899-12-30 00:00:01:080,
0.671,0.239,0.666,1899-12-30 00:00:01:100,
0.666,0.239,0.677,1899-12-30 00:00:01:120,
0.687,0.241,0.68,1899-12-30 00:00:01:140,
0.686,0.245,0.686,1899-12-30 00:00:01:160,
0.67,0.262,0.682,1899-12-30 00:00:01:180,
0.683,0.262,0.679,1899-12-30 00:00:01:200,
0.694,0.249,0.687,1899-12-30 00:00:01:220,
0.677,0.243,0.697,1899-12-30 00:00:01:240,
0.676,0.249,0.687,1899-12-30 00:00:01:260,
0.669,0.256,0.684,1899-12-30 00:00:01:280,
0.686,0.258,0.676,1899-12-30 00:00:01:300,
0.688,0.258,0.677,1899-12-30 00:00:01:320,
0.688,0.267,0.671,1899-12-30 00:00:01:340,
0.709,0.283,0.658,1899-12-30 00:00:01:360,
0.703,0.296,0.652,1899-12-30 00:00:01:380,
0.692,0.31,0.653,1899-12-30 00:00:01:400,
0.704,0.324,0.655,1899-12-30 00:00:01:420,
0.709,0.332,0.661,1899-12-30 00:00:01:440,
0.707,0.351,0.659,1899-12-30 00:00:01:460,
0.753,0.359,0.645,1899-12-30 00:00:01:480,
0.79,0.344,0.647,1899-12-30 00:00:01:500,
0.8,0.335,0.661,1899-12-30 00:00:01:520,
0.781,0.342,0.666,1899-12-30 00:00:01:540,
0.744,0.353,0.664,1899-12-30 00:00:01:560,
0.708,0.375,0.635,1899-12-30 00:00:01:580,
0.673,0.406,0.597,1899-12-30 00:00:01:600,
0.645,0.437,0.573,1899-12-30 00:00:01:620,
0.631,0.464,0.542,1899-12-30 00:00:01:640,
0.659,0.46,0.512,1899-12-30 00:00:01:660,
0.663,0.463,0.492,1899-12-30 00:00:01:680,
0.646,0.485,0.483,1899-12-30 00:00:01:700,
0.688,0.464,0.507,1899-12-30 00:00:01:720,
0.73,0.434,0.532,1899-12-30 00:00:01:740,
0.719,0.447,0.5,1899-12-30 00:00:01:760,
0.674,0.482,0.465,1899-12-30 00:00:01:780,
0.675,0.476,0.456,1899-12-30 00:00:01:800,
0.716,0.439,0.451,1899-12-30 00:00:01:820,
0.728,0.438,0.45,1899-12-30 00:00:01:840,
0.711,0.433,0.49,1899-12-30 00:00:01:860,
0.656,0.461,0.486,1899-12-30 00:00:01:880,
0.623,0.504,0.452,1899-12-30 00:00:01:900,
0.799,0.296,0.472,1899-12-30 00:00:01:920,
0.811,0.303,0.412,1899-12-30 00:00:01:940,
0.779,0.252,0.455,1899-12-30 00:00:01:960,
0.739,0.247,0.455,1899-12-30 00:00:01:980,
0.766,0.218,0.404,1899-12-30 00:00:02:000,
0.823,0.143,0.404,1899-12-30 00:00:02:020,
0.791,0.131,0.446,1899-12-30 00:00:02:040,
0.77,0.128,0.429,1899-12-30 00:00:02:060,
0.771,0.107,0.411,1899-12-30 00:00:02:080,
0.77,0.086,0.404,1899-12-30 00:00:02:100,
0.732,0.063,0.383,1899-12-30 00:00:02:120,
0.767,0.018,0.396,1899-12-30 00:00:02:140,
0.78,0.019,0.386,1899-12-30 00:00:02:160,
0.809,0.02,0.366,1899-12-30 00:00:02:180,
0.779,0.009,0.376,1899-12-30 00:00:02:200,
0.731,-0.024,0.419,1899-12-30 00:00:02:220,
0.712,-0.039,0.439,1899-12-30 00:00:02:240,
0.769,-0.043,0.401,1899-12-30 00:00:02:260,
0.845,-0.049,0.356,1899-12-30 00:00:02:280,
0.888,-0.054,0.347,1899-12-30 00:00:02:300,
0.86,-0.059,0.368,1899-12-30 00:00:02:320,
0.818,-0.083,0.414,1899-12-30 00:00:02:340,
0.815,-0.098,0.434,1899-12-30 00:00:02:360,
0.819,-0.086,0.435,1899-12-30 00:00:02:380,
0.844,-0.106,0.412,1899-12-30 00:00:02:400,
0.84,-0.107,0.408,1899-12-30 00:00:02:420,
0.906,-0.126,0.402,1899-12-30 00:00:02:440,
0.963,-0.126,0.395,1899-12-30 00:00:02:460,
0.93,-0.127,0.422,1899-12-30 00:00:02:480,
0.91,-0.135,0.433,1899-12-30 00:00:02:500,
0.947,-0.152,0.437,1899-12-30 00:00:02:520,
0.975,-0.161,0.442,1899-12-30 00:00:02:540,
0.962,-0.156,0.448,1899-12-30 00:00:02:560,
0.946,-0.156,0.443,1899-12-30 00:00:02:580,
0.95,-0.149,0.43,1899-12-30 00:00:02:600,
0.945,-0.142,0.407,1899-12-30 00:00:02:620,
0.958,-0.168,0.401,1899-12-30 00:00:02:640,
0.963,-0.183,0.412,1899-12-30 00:00:02:660,
0.861,-0.127,0.448,1899-12-30 00:00:02:680,
0.819,-0.116,0.442,1899-12-30 00:00:02:700,
0.885,-0.122,0.395,1899-12-30 00:00:02:720,
0.788,-0.053,0.417,1899-12-30 00:00:02:740,
0.792,-0.052,0.435,1899-12-30 00:00:02:760,
0.818,-0.019,0.434,1899-12-30 00:00:02:780,
0.809,0.024,0.418,1899-12-30 00:00:02:800,
0.793,0.042,0.409,1899-12-30 00:00:02:820,
0.76,0.108,0.437,1899-12-30 00:00:02:840,
0.683,0.157,0.5,1899-12-30 00:00:02:860,
0.704,0.106,0.505,1899-12-30 00:00:02:880,
0.781,0.086,0.46,1899-12-30 00:00:02:900,
0.792,0.156,0.456,1899-12-30 00:00:02:920,
0.72,0.195,0.502,1899-12-30 00:00:02:940,
0.708,0.154,0.536,1899-12-30 00:00:02:960,
0.761,0.123,0.494,1899-12-30 00:00:02:980,
0.821,0.15,0.443,1899-12-30 00:00:03:000,
0.777,0.219,0.466,1899-12-30 00:00:03:020,
0.638,0.251,0.563,1899-12-30 00:00:03:040,
0.61,0.213,0.598,1899-12-30 00:00:03:060,
0.732,0.151,0.51,1899-12-30 00:00:03:080,
0.844,0.178,0.397,1899-12-30 00:00:03:100,
0.756,0.311,0.451,1899-12-30 00:00:03:120,
0.612,0.288,0.576,1899-12-30 00:00:03:140,
0.668,0.206,0.56,1899-12-30 00:00:03:160,
0.851,0.302,0.335,1899-12-30 00:00:03:180,
0.751,0.312,0.433,1899-12-30 00:00:03:200,
0.666,0.264,0.514,1899-12-30 00:00:03:220,
0.7,0.247,0.536,1899-12-30 00:00:03:240,
0.751,0.289,0.528,1899-12-30 00:00:03:260,
0.725,0.294,0.513,1899-12-30 00:00:03:280,
0.673,0.29,0.503,1899-12-30 00:00:03:300,
0.624,0.294,0.588,1899-12-30 00:00:03:320,
0.583,0.294,0.644,1899-12-30 00:00:03:340,
0.625,0.323,0.564,1899-12-30 00:00:03:360,
0.72,0.342,0.482,1899-12-30 00:00:03:380,
0.677,0.392,0.501,1899-12-30 00:00:03:400,
0.537,0.383,0.635,1899-12-30 00:00:03:420,
0.51,0.34,0.713,1899-12-30 00:00:03:440,
0.649,0.323,0.648,1899-12-30 00:00:03:460,
0.755,0.334,0.576,1899-12-30 00:00:03:480,
0.772,0.336,0.577,1899-12-30 00:00:03:500,
0.756,0.335,0.607,1899-12-30 00:00:03:520,
0.688,0.316,0.67,1899-12-30 00:00:03:540,
0.688,0.288,0.666,1899-12-30 00:00:03:560,
0.766,0.285,0.596,1899-12-30 00:00:03:580,
0.771,0.327,0.559,1899-12-30 00:00:03:600,
0.74,0.346,0.587,1899-12-30 00:00:03:620,
0.7,0.344,0.636,1899-12-30 00:00:03:640,
0.678,0.341,0.64,1899-12-30 00:00:03:660,
0.708,0.333,0.608,1899-12-30 00:00:03:680,
0.716,0.332,0.583,1899-12-30 00:00:03:700,
0.675,0.337,0.585,1899-12-30 00:00:03:720,
0.64,0.336,0.617,1899-12-30 00:00:03:740,
0.659,0.345,0.621,1899-12-30 00:00:03:760,
0.69,0.368,0.618,1899-12-30 00:00:03:780,
0.674,0.391,0.652,1899-12-30 00:00:03:800,
0.648,0.415,0.703,1899-12-30 00:00:03:820,
0.691,0.406,0.741,1899-12-30 00:00:03:840,
0.766,0.386,0.754,1899-12-30 00:00:03:860,
0.801,0.374,0.758,1899-12-30 00:00:03:880,
0.772,0.287,0.871,1899-12-30 00:00:03:900,
0.737,0.271,0.882,1899-12-30 00:00:03:920,
0.762,0.274,0.812,1899-12-30 00:00:03:940,
0.811,0.253,0.748,1899-12-30 00:00:03:960,
0.811,0.22,0.734,1899-12-30 00:00:03:980,
0.782,0.196,0.733,1899-12-30 00:00:04:000,
0.755,0.175,0.729,1899-12-30 00:00:04:020,
0.731,0.171,0.719,1899-12-30 00:00:04:040,
0.705,0.164,0.706,1899-12-30 00:00:04:060,
0.643,0.147,0.72,1899-12-30 00:00:04:080,
0.676,0.181,0.676,1899-12-30 00:00:04:100,
0.679,0.181,0.682,1899-12-30 00:00:04:120,
0.681,0.189,0.673,1899-12-30 00:00:04:140,
0.679,0.191,0.671,1899-12-30 00:00:04:160,
0.678,0.187,0.675,1899-12-30 00:00:04:180,
0.692,0.186,0.674,1899-12-30 00:00:04:200,
0.702,0.182,0.677,1899-12-30 00:00:04:220,
0.699,0.179,0.682,1899-12-30 00:00:04:240,
0.701,0.176,0.68,1899-12-30 00:00:04:260,
0.697,0.172,0.682,1899-12-30 00:00:04:280,
0.704,0.167,0.677,1899-12-30 00:00:04:300,
0.7,0.172,0.672,1899-12-30 00:00:04:320,
0.684,0.171,0.68,1899-12-30 00:00:04:340,
0.681,0.175,0.677,1899-12-30 00:00:04:360,
0.679,0.184,0.669,1899-12-30 00:00:04:380,
0.674,0.177,0.673,1899-12-30 00:00:04:400,
0.673,0.173,0.686,1899-12-30 00:00:04:420,
0.688,0.172,0.68,1899-12-30 00:00:04:440,
0.684,0.172,0.683,1899-12-30 00:00:04:460,
0.671,0.18,0.682,1899-12-30 00:00:04:480,
0.67,0.185,0.675,1899-12-30 00:00:04:500,
0.678,0.175,0.683,1899-12-30 00:00:04:520,
0.684,0.174,0.686,1899-12-30 00:00:04:540,
0.691,0.179,0.679,1899-12-30 00:00:04:560,
0.688,0.177,0.68,1899-12-30 00:00:04:580,
0.689,0.176,0.683,1899-12-30 00:00:04:600,
0.689,0.172,0.685,1899-12-30 00:00:04:620,
0.686,0.167,0.684,1899-12-30 00:00:04:640,
0.702,0.167,0.678,1899-12-30 00:00:04:660,
0.697,0.168,0.683,1899-12-30 00:00:04:680,
0.679,0.173,0.685,1899-12-30 00:00:04:700,
0.685,0.168,0.681,1899-12-30 00:00:04:720,
0.69,0.171,0.677,1899-12-30 00:00:04:740,
0.688,0.177,0.671,1899-12-30 00:00:04:760,
0.685,0.175,0.679,1899-12-30 00:00:04:780,
0.688,0.166,0.684,1899-12-30 00:00:04:800,
0.68,0.166,0.69,1899-12-30 00:00:04:820,
0.675,0.173,0.682,1899-12-30 00:00:04:840,
0.678,0.174,0.683,1899-12-30 00:00:04:860,
0.688,0.172,0.675,1899-12-30 00:00:04:880,
0.692,0.173,0.671,1899-12-30 00:00:04:900,
0.689,0.172,0.674,1899-12-30 00:00:04:920,
0.68,0.17,0.682,1899-12-30 00:00:04:940,
0.679,0.167,0.687,1899-12-30 00:00:04:960,
0.683,0.173,0.683,1899-12-30 00:00:04:980,
0.685,0.169,0.679,1899-12-30 00:00:05:000,
0.691,0.168,0.67,1899-12-30 00:00:05:020,
0.695,0.171,0.676,1899-12-30 00:00:05:040,
0.686,0.171,0.679,1900-01-03 23:31:12:000,
0.68,0.171,0.683,1900-01-04 00:00:00:000,
0.692,0.171,0.678,1900-01-04 00:28:48:000,
0.693,0.174,0.677,1900-01-04 00:57:36:000,
------WebKitFormBoundaryu9S9G1KXB3HDtkBs--

Error changing timestamp time data 'wtfwtf' does not match format '%Y-%m-%d %H:%M:%S:%f'
Error changing timestamp time data 'fjdj' does not match format '%Y-%m-%d %H:%M:%S:%f'
Committing 253 records to the database
